### ISSUE TITLE
Mistake on bad-alternative example

### ISSUE THREAD

**Issue by prpcastilho** [angular/angular.io#3086]
On https://angular.io/docs/ts/latest/guide/hierarchical-dependency-injection.html, the example of a bad-alternative has an error with the declarations and bootstrap. 
I think it is changed.
I know it's a bad alternative and should not be used, but I still think it should be correct.

// Don't do this!
@NgModule({
  imports: [
    BrowserModule,
    FormsModule
  ],
  providers: [ HeroesService, RestoreService ],
  **declarations: [ HeroesListComponent ],**
  **bootstrap: [
    HeroesListComponent,
    HeroCardComponent,
    HeroEditorComponent
  ]**
})

**Comment by wardbell** [comment]
I don't think you'll convince us but, as your suggestion is about Angular, not the Angular docs, you must post it as an issue on the angular website.

**Comment by prpcastilho** [comment]
I probably explained myself poorly. I didn't make any suggestion about Angular, but rather I pointed an error existing on Angular docs https://angular.io/docs/ts/latest/guide/hierarchical-dependency-injection.html.
The code above (on my first comment) is not what i suggested to do, but is the error which exists on that page.

The correct solution is:
// Don't do this!
@NgModule({
imports: [
BrowserModule,
FormsModule
],
providers: [ HeroesService, RestoreService ],
**declarations: [
HeroesListComponent,
HeroCardComponent,
HeroEditorComponent
],
bootstrap: 
[ HeroesListComponent ]
}**)

As you write on the live example mention on that page.
Because we only may bootstrap one component, not three components like you have on that example.
I hope I have made me understand better now.
